core/src/main/java/org/infinispan/cache/impl/CacheEncoders.java

```diff
@@ -0,0 +1,77 @@
+package org.infinispan.cache.impl;
+
+import org.infinispan.commons.dataconversion.Encoder;
+import org.infinispan.commons.dataconversion.EncodingUtils;
+import org.infinispan.commons.dataconversion.Wrapper;
+import org.infinispan.marshall.core.EncoderRegistry;
+
+/**
+ * @since 9.2
+ */
+public class CacheEncoders {
+
+   private Encoder keyEncoder;
```

Comment:
These fields should be final, and then you could make them public, instead of using getters.